examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<p>same story as before -- we can probably make this more compact by avoiding redundant symbolic specifications</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
added discussion